internal/governor/governor.go

```diff
@@ -132,12 +135,16 @@ func schedule(r *bufio.Reader, w io.Writer) error {
 
 	line := string(b[:len(b)-1])
 
-	words := strings.SplitN(line, " ", 2)
+	//Forward message to gitrpcd
+	sideband.Write(b)
+
+	words := strings.SplitN(line, " ", 3)
 	switch words[0] {
 	case "continue":
 		return nil
 	case "wait":
 		duration := 1 * time.Second
+		reason := ""
```

Comment:
Would it make sense for the reason here to be "UNKNOWN" by default, similar to the "fail" case?